mysql-test/suite/rocksdb/t/rocksdb_cf_names.test

```diff
@@ -0,0 +1,92 @@
+--source include/have_rocksdb.inc
+
+# Column Family names and name specifications have some pretty strict rules:
```

Comment:
Looks like you wrote the user doc page for this too, make sure Doc engineer sees this

Reply:
That was the intention, there is actually a fair amount more when discussing partitions.